internal/conf/rate.go

```diff
@@ -0,0 +1,93 @@
+package conf
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/didip/tollbooth/v5"
+	"github.com/didip/tollbooth/v5/limiter"
+)
+
+type Rate struct {
+	Events   float64       `json:"events,omitempty"`
+	OverTime time.Duration `json:"over_time,omitempty"`
```

Comment:
for clarification: `Events` keeps track of the number of requests allowed per second while `OverTime` is when the bucket gets refilled right? if so, i would just prefer to call `OverTime` the `ExpirationTTL`

Reply:
As a follow on suggestion, perhaps we can consider renaming events to `AllowedEvents` so that the intentino is clearer

Reply:
I'd prefer to not use "TTL" or "Expire" because it's not about expiring, it's describing allowing number of "Events" occurring "OverTime". 

How that's implemented in rate limiting can vary.